### DISCUSSION TITLE
How can get Galaxy Brain badge

### DISCUSSION THREAD

**Discussion by proarafat** [community/community#18762] [Other Feature Feedback, Questions, & Ideas] [answered]
how can I get galaxy brain badges???
I also answered 2 discussions and those are marked as answered

**Comment by proarafat** [comment]
https://github.com/Schweinepriester/github-profile-achievements
check this link plz

**Reply by proarafat** [reply]
i also ansawered 3 discussion but this badges are not show....

**Reply by kaushikmupadhya** [reply]
You CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.
Hope this answer has helped you, and all the best in getting the achievement! :)

**Reply by Jkeviin** [reply]
> You CANNOT mark your own answers to your own questions. They have to be marked by another user. Discussions should be in a public repository. There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead. Hope this answer has helped you, and all the best in getting the achievement! :)


And how do I fix the bug or error?

**Reply by kaushikmupadhya** [reply]
Opening a discussion and the person who asked the question answering makes no sense right? So obviously I think one cannot achieve this achievement if one answers his own question. I do not think that they might fix this.

**Comment by rilysh** [comment]
Ayo, wtf, you're giving answer of your own question

**Reply by zzkatz** [reply]
Ig, must be an quest to solve something !...

**Comment by mah-shamim** [comment]
https://github.com/Schweinepriester/github-profile-achievements
check this link plz

**Reply by hafijul233** [reply]
Just read through docs on https://github.com/Schweinepriester/github-profile-achievements

**Reply by hafijul233** [reply]
> Just read through docs on https://github.com/Schweinepriester/github-profile-achievements

Galaxy Brain section

**Reply by mah-shamim** [reply]
> 

done

**Comment by hafijul233** [comment]
Just read through docs on https://github.com/Schweinepriester/github-profile-achievements

**Reply by mah-shamim** [reply]
good

**Comment by jofftiquez** [comment]
Answer discussion

**Comment by proarafat** [comment]
But i already answared 3 discussion

**Reply by Andreaa-Dev** [reply]
your answer need to marked as answered by the author

**Reply by williamvietnam** [reply]
You CANNOT mark your own answers to your own questions.

**Comment by MaueDev** [comment, marked as answer]
Bem explicado na documentação https://github.com/Schweinepriester/github-profile-achievements

**Comment by webVueBlog** [comment]
https://github.com/Schweinepriester/github-profile-achievements
请检查此链接

**Comment by Alexander-Kezik** [comment]
I want Galaxy Brain achievement 2,

**Comment by Veercodeprog** [comment]
to achieve the 'Galaxy Brain' achievement, take note of the following:

Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
